<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pbwidget</class>
 <widget class="QWidget" name="pbwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>61</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: rgba(0,0,0,50);
height: 8px;
border-radius: 2px;
}

QSlider::sub-page:horizontal {
background: rgba(0,0,0,200);
border: 1px solid #777;
margin: 1px;
height: 10px;
border-radius: 1px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #FFF, stop:1 #DDD);
border: 1px solid #777;
width: 2px;
margin: -2px 0px;
border-radius: 1px;
}

QSlider::handle:horizontal:hover {
background: #FFF;
border: 1px solid #444;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="prevBtn">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playBtn">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>---</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextBtn">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="playedLbl">
     <property name="text">
      <string>0:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MusicSlider" name="playSlider">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="totalLbl">
     <property name="text">
      <string>0:00</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicSlider</class>
   <extends>QSlider</extends>
   <header>playback/musicslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
